CMakeLists.txt

```diff
@@ -58,7 +58,11 @@ set(LINKER_SCRIPT_TARGET       linker_script_target)
 
 
 define_property(GLOBAL PROPERTY PROPERTY_OUTPUT_FORMAT BRIEF_DOCS " " FULL_DOCS " ")
-set_property(   GLOBAL PROPERTY PROPERTY_OUTPUT_FORMAT elf32-little${ARCH}) # BFD format
+if(CONFIG_ARM64)
+  set_property(   GLOBAL PROPERTY PROPERTY_OUTPUT_FORMAT elf64-littleaarch64) # BFD format
+else()
+  set_property(   GLOBAL PROPERTY PROPERTY_OUTPUT_FORMAT elf32-little${ARCH}) # BFD format
+endif()
```

Comment:
This should be overridden in the arch directory instead to keep the toplevel clean.

Reply:
ACK. Fix coming in V3. Closing to declutter the story. Thanks.